magic-gds-drc/Dockerfile

```
@@ -0,0 +1,47 @@
+# Copyright 2021 Efabless Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update
+RUN apt-get install -y build-essential git curl python3 python3-pip
+
+# Setup Magic
+ENV MAGIC_TAG 8.3.160
+RUN apt-get install -y tcl-dev tk-dev csh
+WORKDIR /tmp/magic
+RUN curl -L https://github.com/RTimothyEdwards/magic/archive/refs/tags/${MAGIC_TAG}.tar.gz | tar -xzC /tmp/magic --strip-components=1
+RUN ./configure && make -j$(nproc) && make install
+WORKDIR /
```

I would put all these in a directory.
```suggestion
WORKDIR /action-drc4gds-magic
```